ansible/vars/aeternity/uat_mon.yml

```diff
@@ -0,0 +1,59 @@
+public_ipv4: "{{ ansible_ec2_public_ipv4|default(ansible_ssh_host)|default(ansible_host)|default(inventory_hostname) }}"
+datadog_enabled: yes
+api_base_uri: http://{{ public_ipv4 }}:{{ node_config.http.external.port }}/v2
+configure_peers: false
+seed_peers:
+  - "52.10.46.160"
+  - "18.195.109.60"
+  - "13.250.162.250"
+  - "13.53.161.215"
+
+node_config:
+  peers:
+    - "aenode://pp_QU9CvhAQH56a2kA15tCnWPRJ2srMJW8ZmfbbFTAy7eG4o16Bf@52.10.46.160:3015"
+    - "aenode://pp_2vhFb3HtHd1S7ynbpbFnEdph1tnDXFSfu4NGtq46S2eM5HCdbC@18.195.109.60:3015"
+    - "aenode://pp_27xmgQ4N1E3QwHyoutLtZsHW5DSW4zneQJ3CxT5JbUejxtFuAu@13.250.162.250:3015"
+    - "aenode://pp_DMLqy7Zuhoxe2FzpydyQTgwCJ52wouzxtHWsPGo51XDcxc5c8@13.53.161.215:3015"
+  sync:
+    port: 3015
+
+  http:
+    external:
+      port: 3013
+    internal:
+      port: 3113
+
+  keys:
+    dir: keys
+    peer_password: secret
+
+  chain:
+    persist: true
+    db_path: "./db{{ db_version|mandatory }}"
+    protocol_beneficiaries_enabled: true
```

Comment:
Beneficiaries configuration is not needed anymore since https://github.com/aeternity/aeternity/commit/b2454cdd0064264f47a09dab89dde188367f4709